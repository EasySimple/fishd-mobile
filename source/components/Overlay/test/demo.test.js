import webDemoTest from '../../../tests/demoTest';

webDemoTest('Overlay');
