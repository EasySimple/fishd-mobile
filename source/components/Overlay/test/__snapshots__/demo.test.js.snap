// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Overlay demo correctly 1`] = `
<div>
  <div
    class="components-overlay-demo-basic"
  >
    <div
      class="demo-title"
    >
      显示遮罩
    </div>
    <a
      aria-disabled="false"
      class="fm-button fm-button-default fm-button-normal"
      role="button"
    >
      <span>
        显示遮罩
      </span>
    </a>
    <span />
  </div>
  <div
    class="components-overlay-demo-basic"
  >
    <div
      class="demo-title"
    >
      嵌入内容
    </div>
    <a
      aria-disabled="false"
      class="fm-button fm-button-default fm-button-normal"
      role="button"
    >
      <span>
        嵌入内容
      </span>
    </a>
    <span />
  </div>
</div>
`;
